_posts/2020-01-29-cognitive-maps.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Cognitive Maps - The Science Behind our Brain’s Internal Mapping and Navigation System
+author:
+  display_name: Greg Bunce
+  email: gbunce@utah.gov
+date: 2020-01-29 10:07:46
+categories:
+  - Featured
+tags:
+  - education
+  - maps
+---
+
+![Network Head]({% link images/network_head.png %}){: .flex .flex--center}
+
+I bet you remember where you were on the morning of September 11th, 2001 when you heard the news. I’d venture to say that you also remember the city, state, and room that you were standing in. But why is this? Why is it that certain memories are permanently tied to a geographic location?
+
```

Comment:
I also might suggest that we move the image down a paragraph so the article blurb isn't empty...

![image](https://user-images.githubusercontent.com/325813/73478586-963abd80-4353-11ea-8b90-6825ef42c273.png)

Reply:
good point.  i'll fix it.